invenio.cfg

```diff
@@ -182,7 +182,7 @@ APP_RDM_DEPOSIT_FORM_AUTOCOMPLETE_NAMES = 'search'  # "search_only" or "off"
 # Invenio-RDM-Records
 # ===================
 # See https://inveniordm.docs.cern.ch/customize/dois/
-DATACITE_ENABLED = False
+DATACITE_ENABLED = True
```

Comment:
please revert, this should not be enabled in local, it causes issues if you don't set the credentials